packages/shared/comms/handlers.ts

```diff
@@ -0,0 +1,253 @@
+import { genericAvatarSnapshots, COMMS_PROFILE_TIMEOUT } from 'config'
```

Comment:
I'm assuming this code was moved without changes. If there is a change, please let me know!